tools/free_energy/alchemical_run/gmx_fep.xml

```diff
@@ -0,0 +1,518 @@
+            </param>
+            <when value="default"/>
+            <when value="modify">
+            <param name="scalpha" type="float" value="0.5" label="The soft-core alpha parameter (sc-alpha)" />
+            <param name="scrpower" type="integer" value="6" label="The power of the radial term in the soft-core equation (sc-r-power)" />
+            <param name="scpower" type="integer" value="1" label="The power for lambda in the soft-core function (sc-power)" />
+            <param name="sccoul" type="text" value="no" label="Apply the soft-core free energy interaction transformation to the Columbic interaction of a molecule (sc-coul)? "/>
+            <param name="scsigma" type="float" value="0.3" label="The soft-core sigma" />
+            <param name="couplelambda0" type="text" value="vdw-q" label="Interactions at lambda=0" help="options are vdw-q, vdw, q, or none"/>
+            <param name="couplelambda1" type="text" value="none" label="Interactions at lambda=1" help="options are vdw-q, vdw, q, or none"/>
+            <param name="coupleintramol" type="text" value="no" label="Turn off intra-molecular interactions?" />
+            </when>
+        </conditional>
+        <param name="temperature" type="float" value="300.0" label="Temperature /K" />
+        <param name="pressure" type="float" value="1.0" label="Pressure /bar" />
+        <param name="input_fep" type="select" label="Generate inputs files only?">
+            <option value="no">No</option>
+            <option value="yes">Yes</option>
+        </param>
+    </inputs>
+    <outputs>
+        <data name="fepinpout" format="tar" from_work_dir="input_files.tar" label="Input files"/>
+        <data name="dataout" format="tar" from_work_dir="data.tar" label="TI/FEP data output"/>
+        <data name="trajout" format="tar" from_work_dir="traj.tar" label="Trajectories output"/>
+        <data name="report" format="txt" label="Report"/>
+    </outputs>
+    <tests>
+         <test>
+            <param name="groin" value="morph.gro" ftype="gro"/>
+            <param name="topin" value="morph.top" ftype="top"/>
+            <param name="minstep" value="10"/>
+            <param name="nvtstep" value="50"/>
+            <param name="nptstep" value="50"/>
+            <param name="mdstep" value="2000"/>
+            <param name="seed" value="19880924"/>
+            <param name="cons" value="No"/>
+            <param name="select_path" value="No"/>
+            <param name="dt" value="0.001"/>
+            <param name="path" value="global"/> 
+            <param name="lambda" value="2"/>
+            <param name="fep" value="0.00 0.50 1.00"/>
+            <param name="temperature" value="300.0"/>
+            <param name="fepop" value="default"/>
+            <param name="input_fep" value="No"/>
+            <param name="pressure" value="1.0"/>
+            <output name="dataout" file="TI_FEP_data_output.tar"  ftype="tar" compare="sim_size"/>
```

Comment:
check out https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-tests-test-output-assert-contents, you could use `has_archive_member` and `has_size` here instead of `sim_size`

Reply:
though has_size is only in the dev version so maybe not that one